src/CoreRT.WebAssembly.MonoBridge/dotnet_support.js

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
src/BuildIntegration might be a better spot for this - I think eventually we'll need the `dotnet` build integration to do something with this file.

Reply:
moved to BuildIntegration/WebAssembly,  or you'd rather not have the subdirectory?